packages/hdwallet-keplr/src/cosmos.ts

```diff
@@ -1,6 +1,6 @@
 import { StdTx } from "@cosmjs/amino";
 import { SignerData } from "@cosmjs/stargate";
-import { ChainReference } from "@shapeshiftoss/caip";
+import { CHAIN_REFERENCE } from "@shapeshiftoss/caip";
```

Comment:
Not strictly related to this PR, but build was failing because of this 🤷🏽

Reply:
caip was bumped from `v3.0.0` to `v8.15.0` it looks like, and types changed

Reply:
Yeah we previously had hoisted types rugging us, this is now happy and consistent!
```shell
=> Found "@shapeshiftoss/caip@8.15.0"
info Reasons this module exists
   - "_project_#@shapeshiftoss#hdwallet-keplr" depends on it
   - Hoisted from "_project_#@shapeshiftoss#hdwallet-keplr#@shapeshiftoss#caip"
   - Hoisted from "_project_#@shapeshiftoss#hdwallet-shapeshift-multichain#@shapeshiftoss#metamask-snaps-adapter#@shapeshiftoss#metamask-snaps#@shapeshiftoss#caip"
```